Source/hydro/Castro_hydro.H

```diff
@@ -541,4 +541,50 @@
 /// @param ng       Number of ghost cells
 ///
     void linear_to_hybrid_momentum(amrex::MultiFab& state, int ng = 0);
+
+#endif
+
+#ifdef TRUE_SDC
+
+    Real compute_laplacian(int i, int j, int k, int n,
```

Comment:
The definition should be consistent with the declaration when it comes to e.g. `AMREX_GPU_DEVICE`. Also I think we should always be using `AMREX_GPU_HOST_DEVICE`.